security/aircrack-ng/Portfile

```diff
@@ -50,6 +47,10 @@ depends_lib         port:hwloc \
                     port:sqlite3
 
 patchfiles-append   aircrack_ng_mac.m4.patch
+if {${configure.build_arch} eq "arm64"} {
+    patchfiles-append \
+                    patch-psedo_intrinsics.patch
+}
```

Comment:
Is there any reason not to apply this patch unconditionally?

Applying conditionally based on `build_arch` will not do the right thing when the universal variant is used.

Is this patch known to upstream? Is there an upstream PR or bug report? The URL you mentioned taking this patch from is not upstream.

Reply:
I see the patch was proposed to upstream two years ago with no comment yet:

https://github.com/aircrack-ng/aircrack-ng/issues/1957#issuecomment-834814596

I see the patch was proposed to Homebrew two years ago and they decided not to use it:

https://github.com/Homebrew/homebrew-core/pull/76993

Reply:
Patch is now changed to be applied unconditionally.
I think homebrew didn't take it because they require patches to be upstreamed first? There's also another patch in that PR which the author thought wasn't good enough that he ultimately closed it.
I have tested some of the binaries (especially `aircrack-ng`, with a test pcap file) on M2 and they seem to work fine. What doesn't work (capturing and stuff) doesn't work on x86 either.